lnrpc/signrpc/signer.pb.go

```diff
@@ -376,6 +376,7 @@ func (m *InputScript) GetSigScript() []byte {
 }
 
 type InputScriptResp struct {
+	// / The set of fully valid input scripts requested.
```

Comment:
After following the new instructions I have a clean result on `make rpc`, this line is not part of the output in my setup.

Reply:
Yes, you are right. So strange.... In any case, this is even better. I rebased and removed this. Thanks!